server/src/main/java/org/elasticsearch/monitor/fs/FsInfo.java

```diff
@@ -36,6 +36,10 @@ public static class Path implements Writeable, ToXContentObject {
         long free = -1;
         long available = -1;
 
+        String effectiveLowWatermark = null;
+        String effectiveHighWatermark = null;
+        String effectiveFloodStage = null;
```

Comment:
Let's keep these as numbers of bytes, not strings. We can convert them to percentages when rendering a response if needed.

Reply:
If we want to render thresholds as percent, we also need to save a total disk space in bytes?
For example: `number of watermark bytes / total disk space = x%`